### PR TITLE
Improving django website, adding generic website

### PR DESCRIPTION
**Pull request by gregelin**
Created a generic web site app based on the Django web site app.

Created a basic GitHub repo app and added it to Django web site app and generic web site app.

### CODE REVIEW

File: apps/GitHub-repo/app.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  status: Operational
+  version: 0.1
+  source_url: https://github.com/GovReady/govready-apps-dev/apps/GitHub-repo
+  description:
+    short: |
+      Primary GitHub source code repository
+    long: |
+      Primary GitHub source code repository.
+
+interstitial:
```
Comment (JoshData) [inline apps/GitHub-repo/app.yaml]: I don't think `interstitial` does anything here.

File: apps/GitHub-repo/github_repo.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    Let's document soem reference information about your primary source code repository.
+
+questions:
+
+  - id: repo_url
+    title: Repo URL
+    prompt: |
+      What is URL of the GitHub source code repository?
+    type: url
+    group:
```
Comment (JoshData) [inline apps/GitHub-repo/github_repo.yaml]: group is invalid/ignored in non-app modules.

File: apps/GitHub-repo/github_repo.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  - id: public_status
+    title: Public or private
+    prompt: |
+      Is the repository public or private?
+    type: choice
+    choices:
+    - key: public
+      text: Public
+      help: Anyone can view the souce code repository.
+    - key: service
```
Comment (JoshData) [inline apps/GitHub-repo/github_repo.yaml]: key should be "private"?

File: apps/GitHub-repo/github_repo.yaml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  - id: created_date
+    title: Created date
+    prompt: |
+      Enter the date the repository was created.
+    type: date
+
+  - id: readme_maintenance
+    title: README maintenance
+    prompt: |
+      Does the repository contain an upto date, well-maintained README file?
```
Comment (JoshData) [inline apps/GitHub-repo/github_repo.yaml]: Typo: "up-to-date"

File: apps/POAM-800-53r4/app.yaml [hunk 5]
```diff
@@ -0,0 +1,71 @@
+id: app
```
Comment (JoshData) [inline apps/POAM-800-53r4/app.yaml]: Do we definitely need a new POAM app specifically for 800-53 rev 4?

Also would recommend renaming to POAM-NIST-SP-800-53r4.

File: apps/django-docker-system/app.yaml [hunk 6]
```diff
@@ -47,6 +48,100 @@ questions:
...
+  title: Actvitity
+  group: Tasks
+  type: module-set
+  protocol:
+  - govready.com/apps/compliance/2018/general/activity
+- id: poams
+  title: POAMs
+  group: Tasks
+  type: module-set
+  protocol:
```
Comment (JoshData) [inline apps/django-docker-system/app.yaml:83]: In questions `protocol` is intersective - only apps that apply to NIST 800-53 r4, r5 AND 171 will be allowed, which probably isn't what you want.

**Review by JoshData — CHANGES_REQUESTED** [review]

File: apps/GitHub-repo/app.yaml (hunk 1 above)
Comment (gregelin) [reply apps/GitHub-repo/app.yaml]: We are no longer showing interstitials?

File: apps/GitHub-repo/github_repo.yaml (hunk 2 above)
Comment (gregelin) [reply apps/GitHub-repo/github_repo.yaml]: Fixed.

File: apps/GitHub-repo/github_repo.yaml (hunk 3 above)
Comment (gregelin) [reply apps/GitHub-repo/github_repo.yaml]: Fixed.

File: apps/GitHub-repo/github_repo.yaml (hunk 4 above)
Comment (gregelin) [reply apps/GitHub-repo/github_repo.yaml]: Fx ed.
;-]

File: apps/POAM-800-53r4/app.yaml (hunk 5 above)
Comment (gregelin) [reply apps/POAM-800-53r4/app.yaml]: Yes, we definitely need a POAM for 800-53 rev 4 bc the controls are different from 800-171, 53r4 and r5. We need differences until we can sort out how to support all the controls.

File: apps/GitHub-repo/app.yaml (hunk 1 above)
Comment (JoshData) [reply apps/GitHub-repo/app.yaml]: Removed about a year ago in ac8a7c2b1ba16fd3bd5e581e21af613fd7b3f6da.

File: apps/POAM-800-53r4/app.yaml (hunk 5 above)
Comment (gregelin) [reply apps/POAM-800-53r4/app.yaml]: I think 800-53 implies NIST-SP but I can change name.

File: apps/django-docker-system/app.yaml (hunk 6 above)
Comment (gregelin) [reply apps/django-docker-system/app.yaml:83]: Fixed.

File: apps/GitHub-repo/app.yaml (hunk 1 above)
Comment (gregelin) [reply apps/GitHub-repo/app.yaml]: Removed.

**Review by JoshData — APPROVED** [review]

**Pull request merged**